.travis.yml

```diff
@@ -10,7 +10,8 @@ env:
     - MOLECULE_DISTRO="fedora:30"
     - MOLECULE_DISTRO="fedora:31"
 install:
-  - pip install molecule docker
+  - pip uninstall molecule
```

Comment:
@i386x @richm @tyll is the pip uninstall needed? Isn't `pip install 'molecule<3'` enough?

Reply:
the install command should be enough

Reply:
Thanks!  Removed uninstall.

Reply:
@nhosoi please rebase to split the travis change into a separate commit and squash it with the fixup here: https://github.com/linux-system-roles/logging/pull/87/commits/07e3fa0d7317a29c62eb5d237c41623c0cb97766#diff-354f30a63fb0907d4ad57269548329e3

Reply:
> @nhosoi please rebase to split the travis change into a separate commit and squash it with the fixup here: [07e3fa0#diff-354f30a63fb0907d4ad57269548329e3](https://github.com/linux-system-roles/logging/commit/07e3fa0d7317a29c62eb5d237c41623c0cb97766#diff-354f30a63fb0907d4ad57269548329e3)

@pcahyna, let me clarify your comment.  I was planning to squash 3 commits in this pr/#87 into one and merge.  Instead, are you suggesting to make 2 commits - one for travis changes and another for the rest?  Thanks.

Reply:
@nhosoi sorry, I did not realize that you want to squash all the commits. But yes, I think that the Travis changes should be in a separate commits from the rest. By the way, I think other changes may also stay in separate commits, like renaming the variables and then adjusting the filters - @richm what do you think?

Reply:
@pcahyna, it might be a bit difficult to make that many separate commits...  Travis is isolated, so it's easy.  For others, I'm afraid I may introduce some bugs (again...):  I don't trust myself. :p

Reply:
@nhosoi what I meant, after splitting off the Travis changes, you could leave the three commits as they are otherwise, making 4 commits in total. @sradco @richm would you prefer the existing commits, or all squashed in one except the Travis changes? By the way, you should still address https://github.com/linux-system-roles/logging/pull/87/commits/69140401652cca448b476c923fe7535f5989ea57#r387825151

Reply:
Ah, I see...  Thanks for the clarification, @pcahyna.
+1 to your suggestion.

Reply:
> @sradco @richm would you prefer the existing commits, or all squashed in one except the Travis changes?

It depends. If @sradco needs to cherry pick these changes to the stable branch, then fewer commits is better.  Otherwise, I think 4 commits is fine.

Reply:
@richm, I don't think this pr is a candidate to backport to the stable branch except the travis/molecule change.  So, as long as the travis/molecule is isolated, we are good.

@sradco, do you want me to backport the travis fix to stable-0.1?